rainbow-cake-core/src/main/java/co/zsmb/rainbowcake/base/RainbowCakeViewModel.kt

```diff
@@ -248,11 +251,11 @@ public abstract class RainbowCakeViewModel<VS : Any>(initialState: VS) : ViewMod
         try {
             task()
         } catch (e: CancellationException) {
-            log(logTag, "Job cancelled exception:")
-            log(logTag, e)
+            log(logTag, "Job cancelled exception:", e, LogLevel.ERROR)
+            logError(logTag, e)
```

Comment:
I have let this method `logError`, but with my new changes it's not really useful anymore because we can embed `throwable` directly in the `log` method